pkg/services/object_manager/storagegroup/collect.go

```diff
@@ -52,7 +54,7 @@ func CollectMembers(r objutil.HeadReceiver, cnr cid.ID, members []oid.ID, calcHo
 	if calcHomoHash {
 		sumHash, err := tz.Concat(phyHashes)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("concatenate '%s' checksums of all members: %w", checksum.TZ, err)
```

Comment:
`%s checksum concat:` to make it somewhat shorter?

Reply:
shorter but less clear (what checksum?)

Reply:
`%s` checksum!

Reply:
Likely it can be deduced from upper level. But I'm OK with this one as well, it's just that (with upper layer wrappers) we quickly get very very lengthy strings.

Reply:
i hope we'll never see them! :magic_wand: